src/groups/mqb/mqba/mqba_application.cpp

```diff
@@ -265,12 +265,11 @@ int Application::start(bsl::ostream& errorDescription)
         }
     }
 
-    mqbi::ClusterResources resources;
-
-    resources.d_blobSpPool_p       = &d_blobSpPool;
-    resources.d_bufferFactory_p    = &d_bufferFactory;
-    resources.d_scheduler_p        = d_scheduler_p;
-    resources.d_pushElementsPool_p = &d_pushElementsPool;
+    mqbi::ClusterResources resources(
```

Comment:
👍🏻 

Recording for posterity: one of the things we thought about and could implement in the future if needed is to make the pools lazy-initialized, so they're only created when first needed.  We're not doing this now, but it could be worthwhile.